beekeeper-api/pom.xml

```diff
@@ -36,6 +36,10 @@
       <groupId>org.projectlombok</groupId>
       <artifactId>lombok</artifactId>
     </dependency>
+    <dependency>
+      <groupId>mysql</groupId>
+      <artifactId>mysql-connector-java</artifactId>
```

Comment:
Why this dependency is needed?

Reply:
Because I was getting a 'Driver not found' error in k8s in the beekeeper-api module and it was solved by adding this dependency. For more information look here https://stackoverflow.com/questions/52804228/failed-to-load-driver-class-com-mysql-jdbc-driver